### DISCUSSION TITLE
Filament panel css is not working (it was working previously)

### DISCUSSION THREAD

**Discussion by piyushdongre-dot-com** [filamentphp/filament#12207] [Help]
### Package

Panel builder

### Package Version

v3.2.57

### How can we help you?

Hi,
Everything was working fine, but suddenly after creating a resource, the panel is not rendering properly, css is broken.
I tried npm run dev/build again few times, but its not working. It will be helpful if you can point out potential cause.

laravel-vite-plugin: 1.0.2 

Thanks

**Comment by micraux** [comment]
Hi,
Have you tried clearing the cache?
